descriptors/ModuleDescriptor-template.json

```diff
@@ -417,6 +417,21 @@
         }
       ]
     },
+    {
+      "id": "finance.release-encumbrance",
```

Comment:
It seems to me that this one and `finance.awaiting-payment` can be part of transactions `finance.transactions` interface